docs/samples/hub-state-import-data-sample.json

```diff
@@ -0,0 +1,925 @@
+          "reportedSuccessfulCount": "select deviceId from devices.modules where moduleId = '$edgeAgent' and configurations.[[mydeployment1]].status = 'Applied' and properties.desired.$version = properties.reported.lastDesiredVersion and properties.reported.lastDesiredStatus.code = 200",
+          "targetedCount": ""
+        },
+        "results": {
+          "appliedCount": 0,
+          "reportedFailedCount": 0,
+          "reportedSuccessfulCount": 0,
+          "targetedCount": 0
+        }
+      },
+      "targetCondition": ""
+    },
+    {
+      "content": {
+        "modulesContent": {
+          "$edgeAgent": {
+            "properties.desired.modules.mymodule0": {
+              "env": {
+                "envvar0": {
+                  "value": "envvarval0"
+                }
+              },
+              "restartPolicy": "always",
+              "settings": {
+                "createOptions": "{\"MemorySwap\":0}",
+                "image": "myimageuri"
+              },
+              "status": "running",
+              "type": "docker",
+              "version": "1.0"
+            }
+          },
+          "$edgeHub": {
+            "properties.desired.routes.route0": "from /messages/* INTO $upstream"
+          },
+          "mymodule0": {
+            "properties.desired.settings": {
+              "prop0": "val0"
+            }
+          }
+        }
+      },
+      "createdTimeUtc": "2022-07-22T22:16:02.3200653Z",
+      "etag": "MQ==",
+      "id": "mylayereddeployment1",
+      "labels": null,
+      "lastUpdatedTimeUtc": "2022-07-22T22:16:02.3200653Z",
+      "metrics": {
+        "queries": {
+          "mymetric": "SELECT deviceId FROM devices WHERE properties.reported.lastDesiredStatus.code = 200"
+        },
+        "results": {}
+      },
+      "priority": 1,
+      "schemaVersion": "2.0",
+      "systemMetrics": {
+        "queries": {
+          "appliedCount": "select deviceId from devices.modules where moduleId = '$edgeAgent' and configurations.[[mylayereddeployment1]].status = 'Applied'",
+          "reportedFailedCount": "select deviceId from devices.modules where moduleId = '$edgeAgent' and configurations.[[mylayereddeployment1]].status = 'Applied' and properties.desired.$version = properties.reported.lastDesiredVersion and properties.reported.lastDesiredStatus.code != 200",
+          "reportedSuccessfulCount": "select deviceId from devices.modules where moduleId = '$edgeAgent' and configurations.[[mylayereddeployment1]].status = 'Applied' and properties.desired.$version = properties.reported.lastDesiredVersion and properties.reported.lastDesiredStatus.code = 200",
+          "targetedCount": ""
+        },
+        "results": {
+          "appliedCount": 0,
+          "reportedFailedCount": 0,
+          "reportedSuccessfulCount": 0,
+          "targetedCount": 0
+        }
+      },
+      "targetCondition": ""
+    }
+  ],
+  "modules": {
+    "device1-child": [],
```

Comment:
I don't see the benefit of modules being key'd by device Id. The device Id is already part of a module identity. This adds significant bloat - imagine a hub with 1 million devices and no modules. We'd have 2 million device Id entries representing the 1 million devices.